Dizzy/Screens/Login/LoginVC.swift

```diff
@@ -100,23 +100,24 @@ final class LoginVC: UIViewController, LoadingContainer, PopupPresenter, CardVC
         }
     }
     
-    private func layoutAppInfosView() {
-        appInfosView.snp.makeConstraints { appInfosView in
-            appInfosView.top.equalTo(loginSelectionView.snp.bottom)
-            appInfosView.leading.trailing.equalToSuperview()
-        }
-    }
-    
     private func layoutDizzyLogo() {
         dizzyLogoImageView.snp.makeConstraints { dizzyLogoImageView in
-            dizzyLogoImageView.top.equalTo(appInfosView.snp.bottom).offset(2 * Metrics.doublePadding)
+            dizzyLogoImageView.top.equalTo(loginSelectionView.snp.bottom).offset(Metrics.doublePadding)
+            dizzyLogoImageView.bottom.equalTo(appInfosView.snp.top).offset(-Metrics.doublePadding)
             dizzyLogoImageView.leading.trailing.equalToSuperview()
         }
     }
     
+    private func layoutAppInfosView() {
+        appInfosView.snp.makeConstraints { appInfosView in
+            appInfosView.top.equalTo(dizzyLogoImageView.snp.bottom).offset(Metrics.doublePadding)
```

Comment:
you dont need to set the top here because you already set it when you set the bottom of `dizzyLogoImageView`

Reply:
fixed